### PR TITLE
ctapipe-quickstart tool

### PR DESCRIPTION
**Pull request by kosack**
adds a simple tool to generate a working directory containing the latest configuration files, since they are no longer available in ctapipe/examples.  You can specify contact info using the command-line options, or if you don't, it prompts the user for the relevant info.  The contact info is then filled into the config files that are generated.

```
% ctapipe-quickstart --workdir "ProdX"
Enter your contact name: Some User
Enter your contact email: me@myemail.com
Enter your organization: CERN
Generated examples in ./ProdX

% ls ProdX

README.md
stage1_config.json
stage2_config.json
training_config.json

% cat ProdX/stage1_config.json 
```

```json
{
    "DataWriter": {
        "Contact": {
            "name": "Some User",
            "email": "me@myemail.com",
            "organization": "CERN"
        },
        "overwrite": false,
        "write_images": true,
        "write_parameters": true,
        "write_stereo_shower": false,
        "write_mono_shower": false,
        "transform_image": true,
        "transform_peak_time": true
    },
    "CameraCalibrator": {
        "image_extractor_type": "NeighborPeakWindowSum"
    },
    "ImageProcessor": {
        "image_cleaner_type": "TailcutsImageCleaner",
        "TailcutsImageCleaner": {
            "picture_threshold_pe": [
                ["type", "*", 10.0],
...
```

### CODE REVIEW

**Comment by HealthyPear** [comment]
we could use this to do what I do in the interface for protopipe...

https://github.com/HealthyPear/protopipe-grid-interface/blob/master/create_analysis_tree.py

**Comment by kosack** [comment]
> we could use this to do what I do in the interface for protopipe...

Yes that's the idea: in the future it would create a full directory structure and fill in some variables in the config files, which are used as templates.  For now, I wanted to start simply just to be able to run a basic analysis, but we can definitely extend this to be more fancy.

Just noticed I need a test... I'll add one

**Comment by kosack** [comment]
added one new feature: the contact information is now filled in from user options (prompted if the user forgets it), so the resulting config file has the correct contact name, email, and organization.

**Comment by HealthyPear** [comment]
> Though if you want, we could just copy your code into it now (or leave that for a future PR?)

I'll check and test it

**Comment by kosack** [comment]
Since I'm away, I think let's just start simple with this one, and not yet add the feature to generate the full directory structure.  That can be a PR we add later, when @HealthyPear  start to merge the features of protopipe (and maybe it's not needed to be as fancy as protopipe here)

File: ctapipe/tools/quickstart.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    name = "ctapipe-quickstart"
+    description = __doc__
+    examples = """
+    To be prompted for contact info:
+
+        ctapipe-quickstart --workdir MyProduction
+
+    Or specify it all in the command-line:
+
+        ctapipe-quickstart --name "my name" --contact "me@thing.com" --org "My Organization" --workdir Work
```
[the remark this answers is not in the document]
Comment (kosack) [reply ctapipe/tools/quickstart.py]: done, thanks!

**Review by HealthyPear — APPROVED** [review]

**Review by LukasNickel — APPROVED** [review]

**Pull request merged**